packages/orbit-components/src/Radio/index.tsx

```diff
@@ -166,6 +166,7 @@ StyledInput.defaultProps = {
   theme: defaultTheme,
 };
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Instead of this, can't we add the `_` to the unused vars? Same for other components

Reply:
yep, it can, it does not make sense to me as it still soon all will be rewritten to Tailwind

Reply:
Alright, non-blocking